examples/misc/lib/effective_dart/usage_good.dart

```diff
@@ -261,14 +261,51 @@ class Box1 {
 
 //----------------------------------------------------------------------------
 
+class Point {
+  Point(this.x, this.y);
+  int x;
+  int y;
+}
```

Comment:
We don't need to define a fourth Point class in this file. Options:

- If you change the `x` getter and setter below from `int` to `num` then math's `Point` class will do.
- Otherwise, `Treasure` and `C` can use `Point2` (the numeric suffix will be stripped away when the code excerpt is extracted into the prose).